lib/aws/stack.ts

```diff
@@ -172,6 +174,101 @@ export class AWSStack extends cdk.Stack {
         ec2.Peer.ipv4("0.0.0.0/0"),
         ec2.Port.tcp(22),
       );
+      const kms_key = new kms.Key(this, "hyperswitch-ssm-kms-key", {
+        removalPolicy: cdk.RemovalPolicy.DESTROY,
+        pendingWindow: cdk.Duration.days(7),
+        keyUsage: kms.KeyUsage.ENCRYPT_DECRYPT,
+        keySpec: kms.KeySpec.SYMMETRIC_DEFAULT,
+        alias: "alias/hyperswitch-ssm-kms-key",
+        description: "KMS key for encrypting the objects in an S3 bucket",
+        enableKeyRotation: false,
+      });
+
+      const external_jump_role = external_jump.getInstance().role;
+      const ext_jump_policy = new iam.PolicyDocument({
+        statements: [
+          new iam.PolicyStatement({
+            actions: [
+              "ssmmessages:CreateControlChannel",
+              "ssmmessages:CreateDataChannel",
+              "ssmmessages:OpenControlChannel",
+              "ssmmessages:OpenDataChannel",
+              "ssm:UpdateInstanceInformation"
+            ],
+            resources: ["*"],
+          }),
+          new iam.PolicyStatement({
+            actions: [
+              "logs:CreateLogStream",
+              "logs:PutLogEvents",
+              "logs:DescribeLogGroups",
+              "logs:DescribeLogStreams"
+            ],
+            resources: ["*"],
+          }),
+          new iam.PolicyStatement({
+            actions: [
+              "s3:GetEncryptionConfiguration"
+            ],
+            resources: ["*"],
+          }),
+          new iam.PolicyStatement({
+            actions: [
+              "kms:Decrypt"
+            ],
+            resources: [kms_key.keyArn],
+          }),
+          new iam.PolicyStatement({
+            actions: [
+              "kms:GenerateDataKey"
+            ],
+            resources: ["*"],
+          }),
+
+        ]
+      });
+      external_jump_role.attachInlinePolicy(
+        new iam.Policy(this, "SessionManagerPolicies", {
+          document: ext_jump_policy,
+        }),
+      );
```

Comment:
Let us create a customer managed policy and attach it to the role, instead of inline policies